server/notebooks/views.py

```diff
@@ -23,6 +23,15 @@ def notebook_view(request, pk):
     })
 
 
+def revision_view(request, nb, pk):
```

Comment:
I think it would be better to make revision a possible parameter for the notebook_view above (and use the latest if it isn't specified).

e.g. https://extremely-alpha.iodide.io/notebooks/1234?revision=123

You can get the revision parameter via the HTTP GET dictionary https://docs.djangoproject.com/en/2.1/ref/request-response/#django.http.HttpRequest.GET

Reply:
Good point!